src/lib/utils/ArtworkInquiry/ArtworkInquiryStore.tsx

```diff
@@ -37,26 +39,23 @@ export const reducer = (
 
     case "selectInquiryQuestion":
       const { isChecked, ...payloadQuestion } = action.payload
-
-      if (isChecked) {
-        inquiryState.inquiryQuestions.push(payloadQuestion)
-      } else {
-        remove(inquiryState.inquiryQuestions, (question) => question.questionID === payloadQuestion.questionID)
-      }
+      const { inquiryQuestions } = inquiryState
```

Comment:
```suggestion
      const { inquiryQuestions = [] } = inquiryState
```

This'll add a default value to inquiryQuestions just in case some wonkiness has happened and the questions are empty.

Reply:
i guess you did actually